specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/PotentialOutages_List.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "nextLink": "string",
+        "value": [
+          {
+            "outageId": "10000064644594",
```

Comment:
same here, each item in the array should be a resource

Reply:
those items will be wrapped inside a "resource". it's addressed in another PR